docs/api/datetime-button.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "ion-datetime-button"
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+import Props from '@site/static/auto-generated/datetime-button/props.md';
+import Events from '@site/static/auto-generated/datetime-button/events.md';
+import Methods from '@site/static/auto-generated/datetime-button/methods.md';
+import Parts from '@site/static/auto-generated/datetime-button/parts.md';
+import CustomProps from '@site/static/auto-generated/datetime-button/custom-props.md';
+import Slots from '@site/static/auto-generated/datetime-button/slots.md';
+
+<head>
+  <title>ion-datetime-button: Ionic API Input for interacting with Datetime picker</title>
+  <meta name="description" content="Datetime button links with a datetime instance to easily a datetime in a popover, modal, and more." />
+</head>
+
+import EncapsulationPill from '@components/page/api/EncapsulationPill';
+import APITOCInline from '@components/page/api/APITOCInline';
+
+<EncapsulationPill type="shadow" />
+
+Datetime Button links with a [Datetime](./datetime) component to display the formatted date and time. It also provides buttons to present the datetime in a modal, popover, and more.
+
+## Overview
+
+Datetime Button should be used when space is constrained. This component displays buttons which show the current date and time values. When the buttons are tapped, the date or time pickers open in the overlay.
+
+When using Datetime Button with a JavaScript framework such as Angular, React, or Vue be sure to use the [keepContentsMounted property on ion-modal](./modal#keepcontentsmounted) or the [keepContentsMounted property on ion-popover](./popover#keepcontentsmounted). This allows the linked datetime instance to be mounted even if the overlay has not been presented yet.
+
+## Basic Usage
+
+import Basic from '@site/static/usage/datetime-button/basic/index.md';
+
+<Basic />
+
+## Localization
+
+The localized text on `ion-datetime-button` is determined by the `locale` property on the associated `ion-datetime` instance. See [Datetime Localization](./datetime#localization) for more details.
+
+## Usage with Modal
+
+`ion-datetime-button` must be associated with a mounted `ion-datetime` instance. As a result, [Inline Modals](./modal#inline-modals-recommended) with the `keepContentsMounted` property set to `true` must be used.
+
+## Usage with Popover
+
+`ion-datetime-button` must be associated with a mounted `ion-datetime` instance. As a result, [Inline Popovers](./popover#inline-popovers) with the `keepContentsMounted` property set to `true` must be used.
```

Comment:
Would it be possible to merge these pieces together, since they're almost identical? I'm debating on whether they're even needed at all since this info is also mentioned in Overview. Also worth noting that separate playgrounds for both modal and popover would be partially redundant, since the Basic playground also shows how to use it with a modal. 🤔